website/snippets/_cloud-environments-info.md

```diff
@@ -28,11 +28,33 @@ Both development and deployment environments have a section called **General Set
 | Default to Custom Branch | ☑️ | Determines whether to use a branch other than the repository’s default  | See below |
 | Custom Branch | dev | Custom Branch name | See below |
 
-:::note About dbt version
+### About versions in dbt Cloud
```

Comment:
love this idea of a new section! 

i'm a little familiar with this content. sugg : 
- make it an H2 instead of an H3
- move this new section to a higher location on the page, above the "Common environment settings" H2 section